test/inductor/test_compiled_autograd.py

```diff
@@ -2102,6 +2102,33 @@ def forward(model, x):
 
         self.assertTrue("CompiledFunctionBackward0" in logs.getvalue())
 
+    def test_verbose_logs_aot_primals(self):
+        torch._logging.set_logs(compiled_autograd_verbose=True)
+
+        def fn():
+            @torch.compile
+            def f(x):
+                tmp1 = x.sin()
+                tmp2 = x.cos()
+                torch._dynamo.graph_break()
+                return tmp1.sin() + tmp2.cos()
+
+            x = torch.randn(4, requires_grad=True)
+            out = f(x)
+            out.sum().backward()
+            yield x.grad
+
+        logs, ctx = logs_to_string(
+            torch._dynamo.compiled_autograd.__name__, "compiled_autograd_verbose"
```

Comment:
Curious does it only rename under `compiled_autograd_verbose`? (wonder if any downside to always rename regardless of verbose or not)

Reply:
the concern is generally overhead, but in this case it is not on the hot path, so i can move it in a followup refactor